### DISCUSSION TITLE
FTP - can see files but not upload or download

### DISCUSSION THREAD

**Discussion by adambendror** [iterate-ch/cyberduck#14454] [Q&A]
Hi all, 

I am running macOS 10.15.7 and Cyberduck 8.5.7. 

I have a server running that I am connecting to over the internet through FTP. In the past Cyberduck has worked perfectly for me but recently I am able to see the contents of the folder that I connect to but I cannot download or upload anything to it. I am able to create new folders using Cyberduck through. 

I have tried changing the read write permissions of the folder as well as turning the firewall off on my mac. 

Please help
Adam

**Comment by LAMepelle** [comment]
Same here. Files that are changed via an external editor are saved but not uploaded.

Thanks!

**Comment by dkocher** [comment]
> Same here. Files that are changed via an external editor are saved but not uploaded.
> 
> Thanks!

That sounds like a different issue.

**Comment by dkocher** [comment]
@adambendror I am moving this to our Q&A forum. Please provide the error message at least to allow others to help.
